### ISSUE TITLE
Unable to locate element after opening React Portal modal

### ISSUE THREAD

**Issue by ekrchan** [chromedp/chromedp#272]
After clicking an element that opens a React Portal modal, I'm attempting to wait on an element inside of the portal div:

```
	if err := s.C.Run(ctxt, cdp.Tasks{
		cdp.Click(hamburgerMenu),
		cdp.WaitVisible(sideBar)}); err != nil {
		return fmt.Errorf("Could not open hamburger menu: %v", err)
	}
```

Chromedp appears to find the sideBar by CSS Selector, but Runtime.evaluate is returning false and it retries the DOM.performSearch. Do I need to do something else to handle the opening of the modal?

Console output below:
```
2019/01/23 17:53:22 <- {"id":249,"method":"DOM.performSearch","params":{"query":".px-sidebar"}}
2019/01/23 17:53:22 -> {"id":249,"result":{"searchId":"1000020452.238","resultCount":1}}
2019/01/23 17:53:22 <- {"id":250,"method":"DOM.getSearchResults","params":{"searchId":"1000020452.238","fromIndex":0,"toIndex":1}}
2019/01/23 17:53:22 -> {"id":250,"result":{"nodeIds":[269]}}
2019/01/23 17:53:22 -> {"method":"DOM.attributeModified","params":{"nodeId":266,"name":"class","value":"portal px-portal enter-active"}}
2019/01/23 17:53:22 <- {"id":251,"method":"DOM.getBoxModel","params":{"nodeId":269}}
2019/01/23 17:53:22 -> {"method":"DOM.attributeModified","params":{"nodeId":266,"name":"class","value":"portal px-portal"}}
2019/01/23 17:53:22 -> {"id":251,"result":{"model":{"content":[0,0,300,0,300,932,0,932],"padding":[0,0,300,0,300,932,0,932],"border":[0,0,300,0,300,932,0,932],"margin":[0,0,300,0,300,932,0,932],"width":300,"height":932}}}
2019/01/23 17:53:22 <- {"id":252,"method":"Runtime.evaluate","params":{"expression":"(function(a) {\n\t\treturn a[0].offsetParent !== null;\n\t})($x('/html[1]/body[1]/div[1]/div[1]/div[1]/div[1]/div[1]/div[1]/div[8]/div[1]/div[2]'))","objectGroup":"console","includeCommandLineAPI":true,"returnByValue":true}}
2019/01/23 17:53:22 -> {"id":252,"result":{"result":{"type":"boolean","value":false}}}
```

**Comment by mvdan** [comment]
Thanks for opening this issue. Could you check if this issue still occurs with the latest chromedp version from master?

**Comment by mvdan** [comment]
Closing for now. Please feel free to reopen or file a new issue if you still experience this bug.

**Comment by ekrchan** [comment]
@mvdan I've tried on `ac47d6ba0e04` and I'm still experiencing this bug. Should I open a new issue? I don't have the ability to reopen this issue.

**Comment by mvdan** [comment]
Apologies, I thought authors were able to reopen issues. Could you please provide a full program that reproduces this bug? If you can't disclose the website that caused this issue, perhaps try reproducing it with a public or static site.

**Comment by ekrchan** [comment]
@mvdan I've dug in a bit more and it's only the sideBar element that still doesn't work; I can select and wait on elements inside of that div. Interacting with other React Portal modals also works. I'm going to close this issue. Thanks for your help!
